pipeline/02-assess.R

```diff
@@ -19,8 +19,7 @@ rsn_prefix <- gsub("_tot", "", params$ratio_study$near_column)
 # Load the training data to use as a source of sales. These will be attached to
 # PIN-level output (for comparison) and used as the basis for a sales ratio
 # analysis on the assessment data
-sales_data <- read_parquet(paths$input$training$local) %>%
-  filter(!sv_is_outlier)
+sales_data <- read_parquet(paths$input$training$local)
```

Comment:
Here we're keeping outlier sales in the data so they can be _included_ in desk review and _excluded_ from the ratio stats.